scripts/aws_clear_instance.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Removing application and dependencies"
+
+if [ -d "/home/notify-app/notifications-api" ]; then
+    # Remove and re-create the directory
+    rm -rf /home/notify-app/notifications-api
+    mkdir -vp /home/notify-app/notifications-api
+    # Remove installed py3 packages
+    pip3 freeze | xargs pip3 uninstall -y
```

Comment:
`pip uninstall -r requirements.txt` won't uninstall, for example, `werkzeug`, a dependency of flask. if flask upgrades its werkzeug version we'll have the old wheel kicking about, so same problem

think virtualenv is a goer tho